NorthstarDLL/squirrel/squirrel.cpp

```diff
@@ -200,12 +200,55 @@ template <ScriptContext context> void SquirrelManager<context>::VMCreated(CSquir
 
 			if (dependency.Name == pair.second)
 			{
+				std::string v = dependency.Version;
+				std::vector<int> semver;
+				size_t last = 0;
+				size_t next = 0;
+
+				while ((next = v.find(".", last)) != std::string::npos)
+				{
+					std::string sub = v.substr(last, next - last);
+					if (std::all_of(sub.begin(), sub.end(), ::isdigit))
+						semver.push_back(stoi(sub));
+					else
+						semver.push_back(-1);
+					last = next + 1;
+				}
+				if (last < v.size())
+				{
+					std::string sub = v.substr(last);
+					if (std::all_of(sub.begin(), sub.end(), ::isdigit))
+						semver.push_back(stoi(sub));
+					else
+						semver.push_back(-1);
+				}
+				else
+					semver.push_back(-1);
+
+				last = 0;
+				next = 0;
```

Comment:
idk if this is needed but it looked like without this the launcher crashes sometimes when reloading mods